package.json

```diff
@@ -84,6 +84,7 @@
     "form-data": "3.0.0",
     "husky": "4.2.5",
     "jasmine": "3.5.0",
+    "jasmine-spec-reporter": "7.0.0",
```

Comment:
Strange that this shows up here as a change, didn't we just add this with the previous PRs?

Reply:
I merged fix-lts-ci-7659 into fix-graphql-deps. Did a reset on fix-lts-ci-7659 to remove my latest changes. Isn't `"jasmine-spec-reporter": "7.0.0"` part of fix-lts-ci-7659? (ps: are you you from Vienna? I'm in Mauer, 1230...)

Reply:
Oh, now I'm confused, where did the graphql changes go???

Reply:
Maybe you want to force push this branch to the latest commit on the 4.x branch and re-do the graphql changes, if they were not too many?

Reply:
ok, yeah that might be best!